src/features/projectsV2/ProjectsContext.tsx

```diff
@@ -92,6 +96,8 @@ export const ProjectsProvider: FC<ProjectsProviderProps> = ({
   const [hoveredPlantLocation, setHoveredPlantLocation] =
     useState<PlantLocation | null>(null);
   const [selectedSite, setSelectedSite] = useState<number | null>(null);
+  const [selectedIntervention, setSelectedIntervention] = useState<string>('all');
+  const [disableInterventionMenu, setDisableInterventionMenu] = useState<boolean>(false);
```

Comment:
The explicit type annotations (<string> and <boolean>) are not necessary here because TypeScript can automatically `infer the types based on the initial values provided`.

To make the code more concise, you can omit these explicit type annotations:
const [selectedIntervention, setSelectedIntervention] = useState('all');
const [disableInterventionMenu, setDisableInterventionMenu] = useState(false);